docs/docs/middleware_reference/data.md

```diff
@@ -3,8 +3,10 @@ This middleware parses the specified yaml files and makes their content availabl
 
 ## Options
 Options are a map where the key is the name of the attribute and the value is the path to the data file relative to the `src/data` folder.
-
+#####Example
```

Comment:
Or, if you want this headline to look always the same, bold text will also work.